js/redirections.js

```diff
@@ -52,7 +52,6 @@ let redirections = {
     "/1.2/learn/making-ballerina-programs-configurable/trying-out-configurability":"/page-not-available.html",
     "/1.2/learn/observing-ballerina-programs/observing-your-application-with-choreo/":"/page-not-available.html",
     "/1.2/learn/running-ballerina-programs-in-the-cloud/code-to-cloud-deployment/":"/page-not-available.html",
-    "/1.2/learn/running-ballerina-programs-in-the-cloud/code-to-cloud-samples/":"/page-not-available.html",
```

Comment:
Keep this redirection for 1.2 and add another entry by duplicating this for Swan Lake page removal as follows.
"/learn/running-ballerina-programs-in-the-cloud/code-to-cloud-samples/":"/page-not-available.html",